pkg/controllers/customcluster_controller.go

```diff
@@ -154,13 +148,18 @@ func (r *CustomClusterController) Reconcile(ctx context.Context, req ctrl.Reques
 		}
 		log.Error(err, "failed to find customCluster", "customCluster", req)
 		// Error reading the object - requeue the request.
-		return ctrl.Result{RequeueAfter: RequeueAfter}, err
+		return ctrl.Result{}, err
 	}
 	log = log.WithValues("customCluster", klog.KObj(customCluster))
 	ctx = ctrl.LoggerInto(ctx, log)
 	// ensure customCluster status no nil
 	if len(customCluster.Status.Phase) == 0 {
 		customCluster.Status.Phase = v1alpha1.PendingPhase
+		// using patchHelper.Patch() to update status need ensure "isConditionsSetter" is true.
+		if err := r.Status().Update(ctx, customCluster); err != nil {
```

Comment:
what do you mean

Reply:
This is not needed, my understanding is wrong